### ISSUE TITLE
Add keyboard support for folders inside desktop GUI

### ISSUE THREAD

**Issue by desfero** [cypress-io/cypress#3741]
### Current behavior:
Currently, folders are omitted during keyboard navigation in the desktop GUI because `onClick` event for folders are attached to a non-interactive element (`div`).

### Desired behavior:

It should be possible to interact with folders (collapsing/expanding) with a keyboard.

### Steps to reproduce: (app code and test code)

See the gif below. Folders don't participate in keyboard navigation.

![Mar-18-2019 21-43-16](https://user-images.githubusercontent.com/7031578/54558710-13bd3f80-49c7-11e9-9742-107224ba8016.gif)

**Comment by nuagesdencre** [comment]
I'll take a look at it!

**Comment by Manuel-Suarez-Abascal** [comment]
Hi, @jennifer-shehane I would like to work on this issue. Can you onboard me on how to tackle it? Which is the related file on the codebase?

**Comment by Manuel-Suarez-Abascal** [comment]
@jennifer-shehane @desfero I have tackled this issue. Can I get a review for [this PR,](https://github.com/cypress-io/cypress/pull/15465) please?
